proto/tpm_attestz.proto

```diff
@@ -48,14 +48,20 @@ message AttestRequest {
 // Response from the specified control card containing attestation evidence of
 // its boot state.
 message AttestResponse {
+  message AttestationCert {
+    oneof value {
+      string aik_cert = 1;  // TPM 1.2
+      string oiak_cert = 2; // TPM 2.0
```

Comment:
should we include LAK here as well?

Reply:
We have not defined an enrollz workflow for LAK, I propose we add LAK workflow in a separate PR.